single_header/H3API.hpp

```diff
@@ -18606,8 +18606,10 @@ namespace h3
 	public:
 		/** @brief [132A0] */
 		INT32 turnsSinceLastEnchanterCast[2];
+		/** @brief [132A8] is used to prevent other monster summon */
```

Comment:
Changes to the single_header are never necessary because it's an automatically generated file.